### PR TITLE
Hotfix edit popup

### PR DESCRIPTION
**Pull request by jannessm**
some functionality was not working...

### CODE REVIEW

File: src/app/components/song-event-form/song-event-form.component.ts [hunk 1]
```diff
@@ -49,39 +60,27 @@ export class SongEventFormComponent implements OnInit {
   onSave():void{
     switch(this.type){
       case DATABASES.songs:
+        this.song.books = this.songBooksStr
+          .split(';')
+          .map(Function.prototype.call, String.prototype.trim)
+          .filter((val) => {
+            return /\w/g.test(val);
```
Comment (nilsroesel) [inline src/app/components/song-event-form/song-event-form.component.ts]:
```javascript
.filter(val =>/\w/g.test(val))
```
would be nicer

File: src/app/components/song-event-form/song-event-form.component.ts [hunk 2]
```diff
@@ -49,39 +60,27 @@ export class SongEventFormComponent implements OnInit {
...
+          });
+        console.log(this.song);
```
Comment (nilsroesel) [inline src/app/components/song-event-form/song-event-form.component.ts]: on purpose log?

File: src/app/components/song-event-form/song-event-form.component.ts [hunk 3]
```diff
@@ -49,39 +60,27 @@ export class SongEventFormComponent implements OnInit {
   onSave():void{
     switch(this.type){
       case DATABASES.songs:
+        this.song.books = this.songBooksStr
+          .split(';')
+          .map(Function.prototype.call, String.prototype.trim)
```
Comment (nilsroesel) [inline src/app/components/song-event-form/song-event-form.component.ts]: not really sure, what this mapping should do?!
```typescript
.map(callback: (currentValue, index?, array?) => any, thisArg: Object)
```

File: src/app/components/song-event-form/song-event-form.component.ts (hunk 2 above)
Comment (jannessm) [reply src/app/components/song-event-form/song-event-form.component.ts]: ups

File: src/app/components/song-event-form/song-event-form.component.ts (hunk 3 above)
Comment (jannessm) [reply src/app/components/song-event-form/song-event-form.component.ts]: this trims each entry in the array.

Comment (nilsroesel) [reply src/app/components/song-event-form/song-event-form.component.ts]: then i guess:
```typescript
.map(value => value.trim())
```

Comment (jannessm) [reply src/app/components/song-event-form/song-event-form.component.ts]: same same

**Pull request merged**